repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3917,7 +3919,10 @@ private List<AtlasEntity> updateClassificationText(AtlasClassification classific
                 AtlasEntity entity = null;
                 for (int i = 1; i <= MAX_NUMBER_OF_RETRIES; i++) {
                     try {
-                        entity = instanceConverter.getAndCacheEntity(graphHelper.getGuid(vertex), ENTITY_CHANGE_NOTIFY_IGNORE_RELATIONSHIP_ATTRIBUTES);
+                        AtlasPerfMetrics.MetricRecorder metricRecorder2 = RequestContext.get().startMetricRecord("updateClassificationText.toAtlasEntity");
```

Comment:
I remember previously when I tried to implement this same, question was on performance impact of this on the operation.

 Could we get a impact of this

Reply:
For the context PR for same was: https://github.com/atlanhq/atlas-metastore/pull/446

Reply:
At that time we didn't did much perf tests, also could we modify cache in memory to maintain correct state?